.github/workflows/run-tests.yml

```
@@ -487,12 +488,24 @@ jobs:
           annotate_only: true
           skip_annotations: true
 
+      - name: Get job ID from workflow run
+        id: get_job_id
+        run: |
+          set -e          
+          job_id=$(curl -L \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{secrets.GITHUB_TOKEN}}" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/"${GITHUB_REPOSITORY}"/actions/runs/"${GITHUB_RUN_ID}"/jobs \
```

I'd write this like
```suggestion
            "https://api.github.com/repos/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}/jobs" \
```
what you have is equivalent but the quotes on outside reads better